tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -962,6 +962,18 @@ vlan/test_vlan_ping.py:
 #######################################
 #####             voq             #####
 #######################################
+voq/test_voq_init.py:
+  skip:
+    reason: cisco-8000 platform is not VOQ based thus all VOQ testcases are not applicable to this platform and should be skipped
+    conditions:
+      - "asic_type in ['cisco-8000']"
+
+voq/test_voq_intfs.py:
+  skip:
+    reason: cisco-8000 platform is not VOQ based thus all VOQ testcases are not applicable to this platform and should be skipped
+    conditions:
+      - "asic_type in ['cisco-8000']"
```

Comment:
lets add this as condition: `(VOQ_INBAND_INTERFACE) == 0`. This makes it more generic as this VOQ property. Also ladd this condition for all the VOQ test cases below  also . We don't want to run any VOQ test case if len(VOQ_INBAND_INTERFACE) == 0

Reply:
Done